### PR TITLE
Added BrowserStack logo to README.md on the main page

### PR DESCRIPTION
**Pull request by antdanchenko**
- The logo is added in order to apply for open source [plan](https://www.browserstack.com/open-source?ref=pricing)
- BrowserStack is a solution fro running e2e tests against iOS
- The platform provides real devices instead of emulators on SauceLabs
- Also would be a good improvement for Android e2e tests in future, if the platform is stable

### CODE REVIEW

**Review by mandrigin — APPROVED** [review]

**Review by annadanchenko — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -62,3 +62,6 @@ Feel free to email us at [support@status.im](mailto:support@status.im) or better
 ## License
 
 Licensed under the [Mozilla Public License v2.0](https://github.com/status-im/status-react/blob/develop/LICENSE.md)
+
+## Testing Supported by<br/>
```
Comment (oskarth) [inline README.md]: don't need br in markdown

**Review by oskarth — COMMENTED** [review]
lgtm, see comment (at least afaik?)

File: README.md (hunk 1 above)
Comment (oskarth) [reply README.md]: If you add a space between header and body it should look fine I think

Comment (antdanchenko) [reply README.md]: br has been removed

**Pull request merged**